src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -141,13 +223,36 @@ public TurretSubsystem(RobotStateManager robotStateManager) {
 
     highGearCANcoder.getConfigurator().apply(highGearSensorConfigs);
     lowGearCANcoder.getConfigurator().apply(lowGearSensorConfigs);
+    turretPIDController.setPID(
+        Constants.TurretConstants.TURRET_KP,
+        Constants.TurretConstants.TURRET_KI,
+        Constants.TurretConstants.TURRET_KD);
 
-    simEncoder =
+    simTurretEncoder =
         new SimDeviceSim(
             "CANEncoder:CANCoder (v6)", Constants.TurretConstants.highGearCAN_CODER_ID);
-    simTurretPos = simEncoder.getDouble("rawPositionInput");
+    simTurretPos = simTurretEncoder.getDouble("rawPositionInput");
+
+    zeroTurret();
+    turretPIDController.setIZone(Constants.TurretConstants.TURRET_KIZ);
+
+    // Pitch
+    pitchPIDController =
+        new PIDController(
+            Constants.TurretConstants.PITCH_KP,
+            Constants.TurretConstants.PITCH_KI,
+            Constants.TurretConstants.PITCH_KD);
+    pitchCANcoder =
+        new Encoder(
```

Comment:
Apparently it is unlikely that we will actually be using this type of encoder. We should find out exactly what kind of encoder we will be using, and then use that here. Once that is done, we can find out how to properly simulate the encoder.